### PR TITLE
[TERRA-564] Add subcommand to get Notebook proxyUrl

### PR DESCRIPTION
**Pull request by dexamundsen**
- New  subcommand to get Notebook proxyUrl
- connected test

Manual test - 

```
terra-cli (dexamundsen/terra-564) >> terra notebook launch --name=dex-notebook-516-1
<verified classic view url>

terra-cli (dexamundsen/terra-564) >> terra notebook launch --name=dex-notebook-516-1 --proxy-view=JUPYTERLAB
<verified lab view url>

terra-cli (dexamundsen/terra-564) >> terra notebook stop --name=dex-notebook-516-1
Notebook instance stopped

terra-cli (dexamundsen/terra-564) >> terra notebook launch --name=dex-notebook-516-1
Cannot launch notebook instance: Expected notebook instance status is [InService] but current status is Stopped
```
image

### CODE REVIEW

File: src/main/java/bio/terra/cli/command/notebook/Launch.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+public class Launch extends BaseCommand {
+  private static final String PROXY_URL = "proxy_url";
+  @CommandLine.Mixin NotebookInstance instanceOption;
+  @CommandLine.Mixin WorkspaceOverride workspaceOption;
+  @CommandLine.Mixin Format formatOption;
+
+  @CommandLine.Option(
+      names = "--proxy-view",
+      description = "AWS Sagemaker Notebook view to be launched: ${COMPLETION-CANDIDATES}.",
+      defaultValue = "JUPYTER",
```
Comment (jmczerk) [inline src/main/java/bio/terra/cli/command/notebook/Launch.java:31]: I think the lab view is probably the better default.

**Review by jmczerk — APPROVED** [review]

File: src/main/java/bio/terra/cli/command/notebook/Launch.java (hunk 1 above)
Comment (dexamundsen) [reply src/main/java/bio/terra/cli/command/notebook/Launch.java]: Done

**Comment by rogerwangcs** [comment]
John C is working on an axonserver endpoint to get the proxy url right?

**Comment by dexamundsen** [comment]
> John C is working on an axonserver endpoint to get the proxy url right?

Yes, we discussed using axon and decided to go ahead with implementing it in CLI for 2 reasons
- Axon isnt hooked up in CLI yet - this is a straightforward call to get the proxyUrl, and adding it here will unblock meaningful use of the CLI for AWS notebooks 
- final location for all cloud interactions will be CRL, Axon will also use crl functions. This internal refactoring can be taken as tech debt while currently focusing on user capabilities

**Review by rogerwangcs — APPROVED** [review]

**Review by zloery — APPROVED** [review]

File: .github/workflows/tests-nightly.yml [hunk 2]
```diff
@@ -84,15 +84,15 @@ jobs:
       run: |
         echo "Running integration tests against source code for server: ${{ matrix.testServer }}"
         mkdir -p ~/logs-integration-source
-        ./gradlew runTestsWithTag -PtestTag=integration --scan -Pserver=${{ matrix.testServer }} -PcontextDir=$HOME/logs-integration-source -PquietConsole
+        ./gradlew runTestsWithTag -PtestTag=integration -Pplatform=GCP -Pserver=${{ matrix.testServer }} -PcontextDir=$HOME/logs-integration-source -PquietConsole --scan
```
[the remark this answers is not in the document]
Comment (dexamundsen) [reply .github/workflows/tests-nightly.yml:87]: Fixed

File: src/test/java/unit/AwsSageMakerNotebookControlled.java [hunk 3]
```diff
@@ -126,6 +127,29 @@ void listDescribeResolveReflectCreateDelete() throws IOException {
     assertNotNull(
         resolvedCredentials.get("Expiration"), "get credentials returned expiration date time");
 
+    // `terra notebook launch --name=$name` // classic view
+    TestCommand.runCommandExpectSuccessWithRetries("notebook", "launch", "--name=" + name);
+    JSONObject proxyUrl =
+        TestCommand.runAndGetJsonObjectExpectSuccess("notebook", "launch", "--name=" + name);
```
[the remark this answers is not in the document]
Comment (dexamundsen) [reply src/test/java/unit/AwsSageMakerNotebookControlled.java:133]: Nope, Though it states 'launch' there is no access or change to the notebook at the backend

**Pull request merged**